scripts/swupd-performance/swupdservicesetup.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/bash
+
+set -e
+
+echo "Setup staging directory"
+
+if [ -d "$HOME/staging" ]; then
+	echo "Staging directory already exists" || true
+else
+	dir=$HOME/staging
+	mkdir $HOME/staging
+
+	if [ $? -eq 0 ]; then
+		echo "Staging directory: $dir successfully created!"
+	fi
+fi
+
+echo ""
+echo "Setup performance monitoring script config dir"
+
+if [ -d /etc/swupd_perform_update_config ]; then
+	echo "Config directory already exists" || true
+else
+	mkdir /etc/swupd_perform_update_config
+fi
+
+cp -r stagingfilegen.sh /etc/swupd_perform_update_config/
+
+# Setting the systemd for the swupd-update-test.service
+cat <<EOF > /etc/systemd/system/swupd-update-test.service
```

Comment:
If you create another service this can clash with swupd autoupdate. I would name it swupd-update.service and then we don't need the extra timer. Because if you create another service and autoupdate is on we will lose some logs

Reply:
I did that because for this reason

In a new system(swupd auto-update is turned off):
a.) if we use the same systemd unit file : swupd-update.service, initially it will be masked and it will fail when we try to reload it.May be I can see how to unmask a service.

Actually i felt it would be better that way because in that case, because then the only thing we have to make initially is : Do not keep auto-enable on.

Reply:
maybe I can do this by running umask --now swupdate.service

Reply:
Done